daemon/daemon_unix.go

```diff
@@ -460,3 +461,40 @@ func setupInitLayer(initLayer string) error {
 func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
 	return nwapi.NewHTTPHandler(daemon.netController)
 }
+
+func (daemon *Daemon) RegisterLinks(container *Container, hostConfig *runconfig.HostConfig) error {
+	if hostConfig != nil && hostConfig.Links != nil {
```

Comment:
```go
if hostConfig == nil || hostConfig.Links == nil {
    return nil
}
```

Because the rest of the code is the import stuff, nice to have one less indentation